kubernetes/cray-istio-deploy/values.yaml

```diff
@@ -22,7 +22,7 @@ tag: 1.8.6-cray1-distroless
 
 kubectl:
   image:
-    repository: dtr.dev.cray.com/loftsman/loftsman
+    repository: arti.dev.cray.com/csm-docker-stable-local/loftsman
```

Comment:
The appropriate image to use for `kubectl` is `artifactory.algol60.net/csm-docker/stable/docker-kubectl:1.19.15`.

Reply:
Done